paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -452,7 +454,7 @@ export const getIdsAndPreferences = (): IdsAndPreferences | undefined => {
 };
 
 export const createSeed = async (
-  { proxyHostName }: CreateSeedOptions,
+  { proxyHostName, callback }: CreateSeedOptions,
```

Comment:
I don't know the code of PrebidJS and I am wondering if there is a timeout on the RTD modules. I see some `auctionDelay` in the configuration. If there is no timeout on the callback, my intuition is that PrebidJS flow can be struck indefinitely or if there is a timeout, I think that we waste some time for starting the auction when:
1. there is no ids and preferences in the cookies (see the early return) 
2. for whatever reason we have an exception in the `postJson()` method

See your other PR with the call of `setData` and `onDone`:  https://github.com/openx/Prebid.js/pull/46/commits/2a81e40d2b6d69e6c53e125b6c6c187144ba4113

Two suggestions: 
1. Call the callback in the early return of this function
2. Add a `try`/`catch` (and maybe a `finally`) to be sure that `callback` is called.